### PR TITLE
feat: Made the soft keyboard more efficient according to the orientation and screen size

### PR DESCRIPTION
**Pull request by mustaqmustu**
feat: Made the soft keyboard more efficient according to the orientation and screen size

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
I'm sure you have done a good job on the pull request but make sure you follow the pr guidelines by updating issue number, changes and screen shots if any ! :)

File: app/src/main/res/xml/pref_settings.xml [hunk 1]
```diff
@@ -5,19 +5,22 @@
 
     <PreferenceCategory android:title="Login email">
         <PreferenceScreen
+            android:icon="@drawable/index"
```
Comment (batbrain7) [inline app/src/main/res/xml/pref_settings.xml]: how is this related to the title of the PR,

File: app/src/main/res/xml/pref_settings.xml [hunk 2]
```diff
@@ -5,19 +5,22 @@
...
         <PreferenceScreen
+            android:icon="@drawable/index"             
             android:key="display_email"
             android:title="test email"/>
     </PreferenceCategory>
 
     <PreferenceCategory android:title="@string/server_settings_title">
         <PreferenceScreen
             android:key="Server_Select"
+            android:icon="@drawable/server"
```
Comment (batbrain7) [inline app/src/main/res/xml/pref_settings.xml]: this too

File: app/src/main/res/xml/pref_settings.xml [hunk 3]
```diff
@@ -5,19 +5,22 @@
...
         <PreferenceScreen
             android:key="Server_Select"
+            android:icon="@drawable/server"                          
             android:summary="@string/server_select_summary"
             android:title="@string/server_pref"/>
     </PreferenceCategory>
 
     <PreferenceCategory android:title="@string/settings_title">
         <SwitchPreference
+            android:icon="@drawable/to"
```
Comment (batbrain7) [inline app/src/main/res/xml/pref_settings.xml]: +1

**Comment by iamareebjamal** [comment]
Not even the changes are right. We won't be able to review this PR unless there is at most 1 commit and 3 file changes. At most
